tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -211,6 +211,35 @@ public function userShouldGetANotificationWithMessage(string $user, string $subj
 		);
 	}
 
+	/**
+	 * @Then user :user should get last notification with subject :subject and message:
+	 *
+	 * @param string $user
+	 * @param string $subject
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function userShouldGetLastNotificationWithMessage(string $user, string $subject, TableNode $table):void {
+		$this->userListAllNotifications($user);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200);
+		// user checks the last notifiction message only
+		$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent()->ocs->data;
+		$lastNotification = $responseBody[\sizeof($responseBody) - 1];
```

Comment:
```suggestion
		$lastNotification = end($responseBody);
```
Can we use `end` to get last element of an array?